code/modules/reagents/reagent_containers/watering_can.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/reagent_containers/glass/watering_can
+	name = "watering can"
+	desc = "It's a watering can. It is scientifically proved that using a watering can to simulate rain increases plant happiness!"
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "watering_can"
+	inhand_icon_state = "watering_can"
+	lefthand_file = 'icons/mob/inhands/equipment/hydroponics_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/hydroponics_righthand.dmi'
+	custom_materials = list(/datum/material/iron = 200)
+	w_class = WEIGHT_CLASS_NORMAL
+	volume = 100
+	amount_per_transfer_from_this = 20
+	possible_transfer_amounts = list(20,100)
+
+/obj/item/reagent_containers/glass/watering_can/wood
```

Comment:
Not a priority for this PR but this is a nonsensical item path and someone should really go and remove or refactor the `/glass/` out of the reagent_container path going forward since nearly half of the game's reagent containers use it as a base.